_data/repositories.yml

```
@@ -235,6 +235,9 @@
     - display_name: Commando
       repository: jtbin
       path: pocket/zips/jotego.jtcommnd.zip
+    - display_name: Toki
+      repository: jttoki
```

```suggestion
      repository: jtbin
```

The repository should have been `jtbin`. I fixed this in #672